src/GafferArnold/ArnoldDisplacement.cpp

```diff
@@ -200,7 +205,10 @@ bool ArnoldDisplacement::acceptsInput( const Gaffer::Plug *plug, const Gaffer::P
 	{
 		if( const GafferScene::Shader *shader = runTimeCast<const GafferScene::Shader>( inputPlug->source<Plug>()->node() ) )
 		{
-			return runTimeCast<const ArnoldShader>( shader );
+			return
```

Comment:
I don't think we have an established convention for that, and I wasn't really sure the braces improved clarity on the last one. I stuck 'em in one because I didn't feel like debating it, but if you're not sure I might leave this one. At some point I might investigate clang-format as a way of automating the formatting...